packages/@lwc/style-compiler/src/__tests__/fixtures/import-in-scoped/actual.css

```diff
@@ -0,0 +1 @@
+@import './foo.css';
```

Comment:
Add a valid unit test for scoped style to capture the generated output.

Reply:
Done.